packages/common/index.ts

```diff
@@ -446,3 +446,8 @@ export interface TwilioBody {
   From: string;
   ApiVersion: string;
 }
+
+export interface GetReleaseNotesResponse {
+  releaseNotes: unknown;
```

Comment:
The reason this is an `unknown` type is because we didn't want to install the react-notion library on the server just so we could use the `BlockMapType` that it provides